index.js

```diff
@@ -52,3 +52,34 @@ async function fetchMetadata() {
 document.onreadystatechange = () => {
     if (document.readyState === 'complete') { fetchMetadata(); }
 }
+
```

Comment:
This code is already available in the ConseilJS documentation at https://cryptonomic.github.io/ConseilJS/#/

Reply:
indeed, this part of code should not be included.